packages/create-cloudflare/templates/common/c3.json

```diff
@@ -0,0 +1,15 @@
+{
+	"configVersion": 1,
+	"id": "common",
```

Comment:
not a big deal as this was already the same pre-refactoring, but isn't `common` extremely generic and unhelpful here?

I guess that changing it would be a breaking change unfortunately 😓

Reply:
Yup I agree. Unfortunately yeah, changing the template names is breaking. When we're ready to do a major bump we'll batch up a rename of all the things we've come to dislike 😄